### ISSUE TITLE
Extension capabilities to set a user and follow test session activities

### ISSUE THREAD

**Issue by theredfish** [voyages-sncf-technologies/soda-test-service#10]
It could be interesting to follow the sessions per user. A good way to do that is to use the extension capabilities as defined in the [webdriver W3C recommandation](https://www.w3.org/TR/webdriver1/#dfn-extension-capability).

Example :

```json
{
	"browserName": "firefox",
	"browserVersion": "1234",
	"soda:user": "team-42"
}
```
Could be achieved once #16 is done.

**Comment by theredfish** [comment]
I close this issue since #14 comes with the `soda:user`. A new issue will be opened
